Bixal/move.mil
# Feature/157125459/ppmtool controller

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Address pivotal ticket #157125459
- Merge json_response module into parser module
- Adds missing data parsings
- Adds entitlements endpoint and controller
- Adds ppm estimator entpoint and controller

## Testing

To verify the changes proposed in this pull request…

1. Pull branch
1. Decrypt files into discounts-date.csv files.
1. Parse all files with `make parser`.
1. Call /parser/entitlements and verify there are 22 records
1. Call /parser/ppm_estimate and verify there is a success response
1. Call /parser/weight_calculator and verify there is a success response

## Screenshots

https://cl.ly/1C2L1D361p31
